enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -283,4 +294,13 @@
     @Description( "Maximum number of history files for the security log." )
     public static final Setting<Integer> store_security_log_max_archives =
             setting( "dbms.logs.security.rotation.keep_number", INTEGER, "7", min(1) );
+
+    //=========================================================================
+    // Misc settings
```

Comment:
I tried to introduce this as well the other day :D @craigtaverner may have suggestions.